katportalclient/test/test_client.py

```diff
@@ -590,6 +592,26 @@ def test_schedule_block_detail(self):
         self.assertIn('expected_duration_seconds', sb_valid)
         self.assertIn('state', sb_valid)
 
+    @gen_test
+    def test_sb_ids_by_capture_block(self):
+        """Test with no capture block IDs on a subarray."""
+        capture_block_base_url = self._portal_client.sitemap[
+            'capture_blocks']
+
+        self.mock_http_async_client().fetch.side_effect = mock_async_fetcher(
+            valid_response=r"""
+                {"result":
+                    "[{\"capture_block_id\":\"1556092345\",\"owner\":\"CAM\",\"type\":\"OBSERVATION\",\"sub_nr\":1},
+                      {\"capture_block_id\":\"1556062104\",\"owner\":\"CAM\",\"type\":\"OBSERVATION\",\"sub_nr\":2}
+                     ]"
+                }""",
+            invalid_response="""{"result":null}""",
+            starts_with=capture_block_base_url)
+
+        sb_ids = yield self._portal_client.sb_ids_by_capture_block()
```

Comment:
It seems like this unittest fails on the number of provided arguments. Does the sb_ids_by_capture_block function work correctly when you provide no arguments?

Reply:
You are right, I blindly mimicked a similar test. See latest commit for the fix.